src/microsoft/versions.json

```diff
@@ -133,6 +64,8 @@
     ],
     "version": "1.22.6",
     "revision": "1",
+    "preferredMajor": true,
```

Comment:
Shouldn't preferred major be 1.23?

Reply:
For our MAR images, we recommend/prefer the N-1 major version. (This key in particular means `mcr[...]microsoft/golang:1` will point at `1.22.6-1` rather than `1.23.0-1`.)

But: we're just following what a partner team recommended a long time ago. They also recommended that people should include the major version even if the result is the same as using `1`, hence our readme recommends `1.22-cbl-mariner2.0` rather than `1-cbl-mariner2.0`. Definitely something we can revisit and come up with our own decision. 😄

Reply:
Filed an issue to dig further into this:

* https://github.com/microsoft/go-lab/issues/101